model/src/main/kotlin/OrtResult.kt

```diff
@@ -157,6 +158,47 @@ data class OrtResult(
         return analyzerErrors.zipWithDefault(scannerErrors, emptySet()) { left, right -> left + right }
     }
 
+    /**
+     * Return a map of license findings for each project or package [Identifier]. The license findings for projects are
+     * mapped to a list of [PathExclude]s matching the locations where a license was found. This list is only populated
+     * if all file locations are excluded. The list is empty for all dependency packages, as path excludes are only
+     * applied to the projects.
+     *
+     * If [omitExcluded] is set to true, excluded projects / packages are omitted from the result.
+     */
+    fun collectLicenseFindings(omitExcluded: Boolean = false) =
+            sortedMapOf<Identifier, MutableMap<LicenseFinding, List<PathExclude>>>().also { findings ->
+                val excludes = repository.config.excludes
+
+                scanner?.results?.scanResults?.forEach { result ->
+                    val project = analyzer?.result?.projects?.find { it.id == result.id }
+
+                    if (!omitExcluded || excludes?.isPackageExcluded(result.id, this) != true) {
+                        result.results.flatMap { it.summary.licenseFindings }.forEach { finding ->
+                            val matchingExcludes = mutableSetOf<PathExclude>()
+                            val isExcluded = finding.locations.all { location ->
+                                excludes != null && excludes.paths.any { exclude ->
```

Comment:
`excludes != null &&` can be moved to outside of `all`:

    val isExcluded = excludes != null && ...

Reply:
Done.